res/css/views/toasts/_IncomingCallToast.pcss

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_IncomingCallToast {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    pointer-events: initial; /* restore pointer events so the user can accept/decline */
+
+    .mx_IncomingCallToast_content {
+        display: flex;
+        flex-direction: column;
+        margin-left: 8px;
+        width: 100%;
+
+        .mx_CallEvent_info {
+            margin-bottom: 52px;
+            margin-right: 40px;
+            width: 180px;
+
+            .mx_CallEvent_room {
+                font-weight: bold;
+                font-size: $font-15px;
+                line-height: $font-24px;
+
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+
+                margin-bottom: $spacing-4;
+            }
+
+            .mx_CallEvent_message {
+                margin-bottom: $spacing-4;
+            }
+        }
+    }
+
+    .mx_IncomingCallToast_joinButton {
+        position: absolute;
```

Comment:
Is absolute positioning how other toasts do this, too?

Reply:
Probably not though it seemed the simplest solution in this case, can change it, if you think it would be better to not use absolute positioning

Reply:
I think the layout is likely to break this way, for example if the user changes the font size, so I would suggest not using absolute positioning here, yeah